int16 centerX
int16 centerY
